### PR TITLE
Third batch of CTS test refactorings

### PR DESCRIPTION
**Pull request by clumens**
This is it, the last stuff that was in CTStests.py.  Nothing especially exciting here.

### CODE REVIEW

File: configure.ac [hunk 1]
```diff
@@ -1983,7 +1983,6 @@ CONFIG_FILES_EXEC([agents/ocf/ClusterMon],
                   [cts/cts-fencing],
                   [cts/cts-regression],
                   [cts/cts-scheduler],
-                  [cts/lxc_autogen.sh],
```
Comment (kgaillot) [inline configure.ac:1986]: also move the corresponding .gitignore line to "formerly built"

File: doc/sphinx/Pacemaker_Remote/alternatives.rst [hunk 2]
```diff
@@ -78,14 +78,6 @@ using virtual machines. Key differences:
...
   `libvirt-lxc <http://libvirt.org/drvlxc.html>`_ driver for LXC containers.
 
-* Libvirt XML definitions must be generated for the containers. The
-  ``pacemaker-cts`` package includes a script for this purpose,
-  ``/usr/share/pacemaker/tests/cts/lxc_autogen.sh``. Run it with the
-  ``--help`` option for details on how to use it. It is intended for testing
-  purposes only, and hardcodes various parameters that would need to be set
-  appropriately in real usage. Of course, you can create XML definitions
-  manually, following the appropriate libvirt driver documentation.
-
```
Comment (kgaillot) [inline doc/sphinx/Pacemaker_Remote/alternatives.rst:88]: The first and last sentence are still relevant

File: python/pacemaker/_cts/tests/remotestonithd.py [hunk 3]
```diff
@@ -9,12 +8,24 @@
 
 
 class RemoteStonithd(RemoteDriver):
+    """ A concrete test that fails and fences the node hosting a Pacemaker
+        Remote connection resource
```
Comment (kgaillot) [inline python/pacemaker/_cts/tests/remotestonithd.py:12]: it fails the connection resource but the remote node gets fenced (not the connection host)

**Review by kgaillot — COMMENTED** [review]
yay! looks great

**Comment by clumens** [comment]
Updated to address review.

**Review by kgaillot — APPROVED** [review]

**Pull request merged**